addons/mail/static/src/models/messaging.js

```diff
@@ -170,13 +170,14 @@ registerModel({
             } else {
                 const { args, method, model, kwargs = {} } = params;
                 const { domain, fields, groupBy } = kwargs;
+                delete kwargs.fields;
 
                 const ormService = 'shadow' in options ? this.env.services.orm.silent : this.env.services.orm;
                 switch (method) {
                     case 'create':
                         return ormService.create(model, args[0], kwargs);
                     case 'read':
-                        return ormService.read(model, args[0], args.length > 1 ? args[1] : undefined, kwargs);
+                        return ormService.read(model, args[0], fields , kwargs);
```

Comment:
check on `args.length > 1` should be kept to support both syntax (having fields in args or kwargs), just the `undefined` part should be replaced by `fields`.

Reply:
> check on `args.length > 1` should be kept to support both syntax (having fields in args or kwargs), just the `undefined` part should be replaced by `fields`.

done